bdi_game/src/algorithms/search/searchable.rs

```diff
@@ -0,0 +1,18 @@
+use std::hash::Hash;
+use std::ops::{Add, AddAssign};
+
+pub struct StatesArc<T, C: Ord> {
```

Comment:
What does StatesArc stand for?

Reply:
It represents a connection to the other node because it is not enough to know the other node, you also have to know the cost of getting there.